exporter/exporterhelper/common.go

```diff
@@ -27,6 +27,20 @@ func NewDefaultTimeoutSettings() TimeoutSettings {
 	}
 }
 
+// SampledLoggerSettings configures the logger sampler, which caps the CPU and I/O load of logging while keeping a representative subset of your logs.
+// Its purpose is to balance between the need for comprehensive logging and the potential performance impact of logging too much data.
```

Comment:
Is this sentence related to logger sampler? If it is I would remove it, would only focus describing settings